lib/wallet.py

```diff
@@ -79,6 +79,8 @@ def _(message): return message
 del _
 from .i18n import _
 
+DEFAULT_CONFIRMED_ONLY = False
+
 def relayfee(network):
```

Comment:
Making default value accessible to the Android settings UI.

Reply:
I don't pretend to know what this has to do with settings UI -- presumably in the settings you just.. set the setting.. and not the setting's default.  But if you say you need it -- so be it. Does no harm to be here :)